### ISSUE TITLE
Security error on ActiveMX, devices can not connect to message broker

### ISSUE THREAD

**Issue by leroy0211** [h-mdm/hmdm-server#81]
Hi, 

I got a Security error in the mdm server from all my devices.  They can not connect to the queue due to the following reason: 

```
Failed to add Connection id=ID:hmdm01-41253-1684843098146-3:20, clientId=H0010 due to java.lang.SecurityException: User name [hmdm] or password is invalid.
```

I guess the ActiveMX credentials in the launcher are wrong or somehow. 

I do have installed the mdm server twice, because the first time the installation failed. I have not removed the installation before, and during the installation wizard I've entered all details the same as before. 

HMDM Version: 5.18.2 Community
Launcher version: 5.18
Update helper version: 1.04

I couldn't find any ActiveMX or MQTT parameters in ROOT.xml other than `mqtt.server.uri` (string) and `mqtt.auth` (boolean). But not a parameter where the credentials are stored.

Any direction to point me to, to fix this? Now application updates are not pushed, but only fetched if you press the "fetch updates" button.

**Comment by leroy0211** [comment]
Where do you have to update it? In the launcher app, in the server or both? 

BTW, this was a fresh install, not an upgrade. Does that mean that every fresh install of the community edition doesn't work?

**Comment by h-mdm** [comment]
You must update it at the server. There was a bug in the install/context_template.xml which has been fixed.

**Comment by leroy0211** [comment]
I've double checked, and the `hash.secret` is already at that new value. 

```xml
<!-- A shared secret between mobile app and control panel.
         Don't change this unless you know what you're doing -->
    <Parameter name="hash.secret" value=[********-*#*#**##]"/>
```

**Comment by h-mdm** [comment]
Which is the version of launcher you're using?

**Comment by h-mdm** [comment]
There was an issue in the shared secret on the client part, thank you for the bug report. The issue has been fixed in Headwind MDM launcher 5.19, please upgrade the client.
